crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -340,12 +340,24 @@ impl<'a> TrackedRenderPass<'a> {
     /// Set the rendering viewport to the given [`Camera`](crate::camera::Viewport) [`Viewport`].
     ///
     /// Subsequent draw calls will be projected into that viewport.
-    pub fn set_camera_viewport(&mut self, viewport: &Viewport) {
+    pub fn set_camera_viewport(&mut self, viewport: &Viewport, camera: &ExtractedCamera) {
```

Comment:
I had to change the signature of this function, because I needed camera to calculate absolute values of viewport.
Alternatives would be:
- have caller clone the viewport and calculate those values themselves (or make it a method on `Viewport`)
- "bake in" absolute values into viewport (i.e. only use absolutes in viewport) but tbf I'm not sure when would this have to happen (and it wouldn't be responsive)

Reply:
I definitely don't want the second option; that seems much harder to maintain and responsivity is important.

Reply:
Instead of passing in the entire extracted camera, could you instead just pass in the physical target size as that is the only thing that is used?

Reply:
Yeah, that works too

Reply:
Done